scripts/mochify

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd $(dirname $0)/..
+./node_modules/.bin/mochify ./test/**/*.coffee --transform=coffeeify --extension=.coffee
```

Comment:
Just a note; please do not use shell scripts. They are not cross platform and super error prone.
Use node scripts and write javascript code if necessary.